src/richie/apps/courses/migrations/0013_migrate_data_translated_licence_fields.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.8 on 2020-01-02 13:56
+
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    Licence = apps.get_model("courses", "Licence")
+    LicenceTranslation = apps.get_model("courses", "LicenceTranslation")
+
+    for object in Licence.objects.all():
```

Comment:
object is a reserved word in Python, so maybe call it:
```suggestion
    for licence in Licence.objects.all():
```